src/Runner.Worker/Container/ContainerHooks/HookResponse.cs

```diff
@@ -7,6 +7,7 @@ public class HookResponse
     {
         public ResponseContext Context { get; set; }
         public JToken State { get; set; }
+        public bool Alpine { get; set; }
```

Comment:
I know the ADR says Alpine, but let's keep to `IsAlpine` here, the hook's part is written yet